docs/template/public/less/_kss-content.less

```diff
@@ -163,4 +163,12 @@
             top: 35px;
         }
     }
+
+    ul, ol 
+    {
+        &.dataCards 
+        {
+            padding: 0;
+        }
+    }
```

Comment:
We need to fix this on the cards component itself and not the kss site as it is an issue with the component that need to be fixed where ever is it used. ;) 

pls apply the padding-left: 0; in _data-card.less on .dataCards class